packages/shared/src/components/post/PostContentContainer.tsx

```diff
@@ -7,7 +7,10 @@ interface PostContentContainerProps {
   children: ReactNode;
 }
 
-const BodyContainer = classed('div', 'flex flex-col w-full pb-6');
+const BodyContainer = classed(
+  'div',
+  'flex flex-col w-full pb-6 bg-background-default',
+);
```

Comment:
How did this break just now? Seems global.

Reply:
Yeah, actually quite not sure 😅

Reply:
I will investigate after confirming the fix on prod.

Reply:
fylla!